doc/NGC_rapidsai.md

```diff
@@ -31,10 +32,12 @@ images in order to make it easy to add RAPIDS libraries while maintaining suppor
 
 The RAPIDS images provided by NGC come in two types:
 
-- `base` - contains a RAPIDS environment ready for use.<br>Use this image if you want to use RAPIDS as a part of your pipeline.
-- `runtime` - extends the `base` image by adding a notebook server and example notebooks.<br>Use this image if you want to explore RAPIDS through notebooks and examples.
+- `base` - contains a RAPIDS environment ready for use.
+  - **TIP: Use this image if you want to use RAPIDS as a part of your pipeline.**
+- `runtime` - extends the `base` image by adding a notebook server and example notebooks.
+  - **TIP: Use this image if you want to explore RAPIDS through notebooks and examples.**
 
-For images that contain the full RAPIDS source tree, pre-built with all artifacts in place, and the compiler toolchain, the debugging tools, the headers and the static libraries for RAPIDS development - as well as support for Python 3.7 - refer to the [rapidsai/rapidsai-dev](https://cloud.docker.com/u/rapidsai/repository/docker/rapidsai/rapidsai-dev) repo on DockerHub.
+For images that contain the full RAPIDS source tree, pre-built with all artifacts in place, and the compiler toolchain, the debugging tools, the headers and the static libraries for RAPIDS development - as well as support for Python 3.7 - refer to  [rapidsai/rapidsai](https://cloud.docker.com/u/rapidsai/repository/docker/rapidsai/rapidsai) and [rapidsai/rapidsai-dev](https://cloud.docker.com/u/rapidsai/repository/docker/rapidsai/rapidsai-dev) repos on DockerHub.
```

Comment:
Adding `rapidsai/rapidsai` might be misleading since this paragraph is specifically describing `devel` images. Maybe we need something saying:
"For `base` and `runtime` images with python 3.7 support, refer to [rapidsai/rapidsai] on DockerHub.  For images that contain the full RAPIDS source tree...."

Reply:
I agree I'll split this out to remove any confusion